pkg/chartverifier/report.go

```diff
@@ -46,12 +46,18 @@ type ReportMetadata struct {
 }
 
 type ToolMetadata struct {
-	Version                    string `json:"verifier-version" yaml:"verifier-version"`
-	Profile                    string `json:"profileName" yaml:"profileName"`
-	ChartUri                   string `json:"chart-uri" yaml:"chart-uri"`
-	Digest                     string `json:"digest,omitempty" yaml:"digest,omitempty"`
-	LastCertifiedTimestamp     string `json:"lastCertifiedTimestamp,omitempty" yaml:"lastCertifiedTimestamp,omitempty"`
-	CertifiedOpenShiftVersions string `json:"certifiedOpenShiftVersions,omitempty" yaml:"certifiedOpenShiftVersions,omitempty"`
+	Version                    string  `json:"verifier-version" yaml:"verifier-version"`
+	Profile                    string  `json:"profileName" yaml:"profileName"`
+	ChartUri                   string  `json:"chart-uri" yaml:"chart-uri"`
+	Digest                     string  `json:"digest,omitempty" yaml:"digest"`
```

Comment:
Is there a reason we only `omitempty` for json but not for yaml?

Reply:
Good spot, removed it from both now.